modules/images.js

```diff
@@ -0,0 +1,25 @@
+var Images = "";
+
+function randomIcons(){
+    var icons = ["gallery_icon","channel_icon","video_icon","",""];
+    return icons[Math.floor(Math.random() * icons.length)];
+}
+
+for(var i = 1;i<55;i++){
```

Comment:
I'm trying to understand what is going on here. What does the variable `i` represent?

What are you trying to achieve in general? Can you use your code to make this clear?

Reply:
okay sir

Reply:
@devbash1 this one